presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2025,6 +2025,31 @@ public void testHistogram()
     }
 
     @Test
+    public void testJoinWithLessThanInJoinClause()
+            throws Exception
+    {
+        assertQuery("SELECT n.nationkey, r.regionkey FROM region r, nation n WHERE n.regionkey = r.regionkey AND n.name < r.name");
```

Comment:
Also, can you double check that these queries are "fast" in the H2 "control" database.